### PR TITLE
Fix ops, cleanup ToC, add latest illustrations

### PR DESCRIPTION
**Pull request by indie**
Fix `Op` set versioning that breaks doc, update ToC for ngraph.ai, many updated illustrations and small fixes

### CODE REVIEW

File: doc/sphinx/source/frameworks/overview.rst [hunk 1]
```diff
@@ -0,0 +1,50 @@
+.. frameworks/overview.rst
+
+.. _fw_overview:
+
+Overview
+========
+
+A framework is "supported" with a framework :term:`bridge` that can be written or
+cloned and used to connect to nGraph device backends while maintaining the 
+framework's programmatic or user interface. A `bridge currently exists`_ for the
```
Comment (diyessi) [inline doc/sphinx/source/frameworks/overview.rst]: We also have a PaddlePaddle bridge.

File: doc/sphinx/source/frameworks/validated/list.rst [hunk 2]
```diff
@@ -1,24 +1,23 @@
...
 
-* :ref:`tensorflow_valid`
-* :ref:`mxnet_valid`
-* :ref:`onnx_valid`
-* :ref:`testing_latency`
+Validated workloads
+###################
 
-.. _tensorflow_valid:
+We validated performance [#f1]_ for the following workloads:
```
Comment (diyessi) [inline doc/sphinx/source/frameworks/validated/list.rst]: I think `We have validated ...` sounds better here.

Comment (indie) [reply doc/sphinx/source/frameworks/validated/list.rst]: Done

File: doc/sphinx/source/frameworks/overview.rst (hunk 1 above)
Comment (indie) [reply doc/sphinx/source/frameworks/overview.rst]: Done

**Pull request merged**